src/main/java/io/seqera/tower/cli/commands/DataLinksCmd.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021-2023, Seqera.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.seqera.tower.cli.commands;
+
+import io.seqera.tower.cli.commands.data.links.AddCmd;
+import io.seqera.tower.cli.commands.data.links.DeleteCmd;
+import io.seqera.tower.cli.commands.data.links.ListCmd;
+import io.seqera.tower.cli.commands.data.links.UpdateCmd;
+import io.seqera.tower.cli.commands.data.links.BrowseCmd;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+        name = "data-links",
```

Comment:
are we going with `data-links`? I checked the documentation and there is very little mention of `data-links` nor does there seem to be any in the UI. Sounds to me that `data-links` is an internal concept, do we want to expose it to users of the CLI?

Reply:
Here is the same discussion around that: https://github.com/seqeralabs/tower-cli/issues/381#issuecomment-2110451033